test/cloudformation/iam_cloudformation.yaml

```diff
@@ -249,6 +249,21 @@ Resources:
           - Effect: Allow
             Action: timestream:DescribeEndpoints
             Resource: "*"
+  GithubActionsPolicySecond:
```

Comment:
Why does this need to be a new policy?

Reply:
If we need to make it, can we rename it to be more descriptive?

Reply:
We have run out of character space on the policy, so we will need a second policy to add new permissions